### PR TITLE
Auth mini order table too

### PR DESCRIPTION
**Pull request by iaincrawf**

### CODE REVIEW

**Review by richardiphillips — APPROVED** [review]

File: src/Persistence.LinnApps/ServiceDbContext.cs [hunk 1]
```diff
@@ -298,7 +298,7 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
             var connectionString = $"Data Source={dataSource};User Id={userId};Password={password};";
 
             optionsBuilder.UseOracle(connectionString, options => options.UseOracleSQLCompatibility("11"));
-            optionsBuilder.UseLoggerFactory(MyLoggerFactory);
+            //optionsBuilder.UseLoggerFactory(MyLoggerFactory);
```
Comment (RossStewart) [inline src/Persistence.LinnApps/ServiceDbContext.cs]: Do we want to leave this in for now?

**Review by RossStewart — APPROVED** [review]

File: src/Persistence.LinnApps/ServiceDbContext.cs (hunk 1 above)
Comment (RossStewart) [reply src/Persistence.LinnApps/ServiceDbContext.cs]: Sorry, saw you caught it

**Comment by iaincrawf** [comment]
@RossStewart I think I forgot to merge this if you want to

**Comment by RossStewart** [comment]
> @RossStewart I think I forgot to merge this if you want to

haha! Will do!

**Pull request merged**